src/main/java/org/folio/circulation/domain/RequestAndRelatedRecords.java

```diff
@@ -90,11 +99,15 @@ public RequestQueue getRequestQueue() {
   RequestPolicy getRequestPolicy() { return requestPolicy; }
 
   String getOriginalItemId() {
-    return moveRequestRecord != null ? moveRequestRecord.getOriginalItemId() : null;
+    return isMoveRequest() ? moveRequestRecord.getOriginalItemId() : null;
   }
 
   String getDestinationItemId() {
-    return moveRequestRecord != null ? moveRequestRecord.getDestinationItemId() : null;
+    return isMoveRequest() ? moveRequestRecord.getDestinationItemId() : null;
+  }
+
+  public boolean isMoveRequest() {
```

Comment:
Eventually, I think this and the two properties above, need to come out of the request class. To me they are properties of a workflow not the request itself. 

Doing this likely relies on broader changes to how the domain and workflows are modelled in mod-circulation, and so is out of scope of this change.

Reply:
I will look into other ways to provide if request and related records is operating within a move request.

Reply:
I think it is worth deferring that for another time. There is quite a lot of changes likely needed to make this possible (removing the incidental coupling on JSON for persistence, changing how workflows are modelled etc)